vignettes/skipping.Rmd

```diff
@@ -51,57 +51,66 @@ All reporters show which tests as skipped.
 As of testthat 3.0.0, ProgressReporter (used interactively) and CheckReporter (used inside of `R CMD check`) also display a summary of skips across all tests.
 It looks something like this:
 
-    ── Skipped tests  ───────────────────────────────────────────────────────
-    ● No token (3)
-    ● On CRAN (1)
+```         
+── Skipped tests  ───────────────────────────────────────────────────────
+● No token (3)
+● On CRAN (1)
+```
 
 You should keep an on eye this when developing interactively to make sure that you're not accidentally skipping the wrong things.
 
 ## Helpers
 
 If you find yourself using the same `skip_if()`/`skip_if_not()` expression across multiple tests, it's a good idea to create a helper function.
-This function should start with `skip_` and live somewhere in your `R/` directory.
+This function should start with `skip_` and live in a `test/helper-{something}.R`:
 
 ```{r}
-skip_if_Tuesday <- function() {
-  if (as.POSIXlt(Sys.Date())$wday != 2) {
-    return(invisible(TRUE))
+skip_if_dangerous <- function() {
+  if (!identical(Sys.getenv("DANGER"), "")) {
+    skip("Not run in dangerous environments.")
+  } else {
+    invisible()
```

Comment:
Connecting back to my nitpick comment on the other PR, I find negated conditions sort of mind-bendy. Can we do it the other (more natural) way around?

```suggestion
skip_if_dangerous <- function() {
  if (identical(Sys.getenv("DANGER"), "")) {
    invisible()
  } else {
    skip("Not run in dangerous environments.")
```

Another nice example:

```
skip_if_not_utf8 <- function() {
  skip_if_not(l10n_info()$`UTF-8`)
}
```

Reply:
Let's discuss that more there, because I find the current structure more clear.